products/llm_analytics/frontend/llmAnalyticsTraceLogic.ts

```diff
@@ -190,19 +190,19 @@ export const llmObservabilityTraceLogic = kea<llmObservabilityTraceLogicType>([
 
     listeners(({ actions, values }) => ({
         setIsRenderingMarkdown: ({ isRenderingMarkdown }) => {
-            localStorage.setItem('llm-observability-markdown-rendering', JSON.stringify(isRenderingMarkdown))
+            localStorage.setItem('llm-analytics-markdown-rendering', JSON.stringify(isRenderingMarkdown))
         },
         toggleMarkdownRendering: () => {
-            localStorage.setItem('llm-observability-markdown-rendering', JSON.stringify(values.isRenderingMarkdown))
+            localStorage.setItem('llm-analytics-markdown-rendering', JSON.stringify(values.isRenderingMarkdown))
         },
         setIsRenderingXml: ({ isRenderingXml }) => {
-            localStorage.setItem('llm-observability-xml-rendering', JSON.stringify(isRenderingXml))
+            localStorage.setItem('llm-analytics-xml-rendering', JSON.stringify(isRenderingXml))
         },
         toggleXmlRendering: () => {
-            localStorage.setItem('llm-observability-xml-rendering', JSON.stringify(values.isRenderingXml))
+            localStorage.setItem('llm-analytics-xml-rendering', JSON.stringify(values.isRenderingXml))
         },
         setDisplayOption: ({ displayOption }) => {
-            localStorage.setItem('llm-observability-display-option', JSON.stringify(displayOption))
+            localStorage.setItem('llm-analytics-display-option', JSON.stringify(displayOption))
         },
```

Comment:
Oh wow, this is much more complicated than it needs to be. Kea has built-in `localStorage` permanence. You can check the Revenue/Web Analytics logics to see how we do it in there. TLDR; second parameter on the reducer

Reply:
This is a really great tip, thank you!

@k11kirky 
Look [this commit](https://github.com/PostHog/posthog/pull/37001/commits/1521c469ae6e6e8237e41d342a5c4ba7a77ee949), let's manage localstorage like this from now on!

Resulting keys:
<img width="407" height="104" alt="image" src="https://github.com/user-attachments/assets/9e3f007e-40ac-4f59-8225-09d387899090" />